korman/exporter/physics.py

```diff
@@ -212,11 +212,11 @@ def _iter_object_tree(bo, stop_at_subworld):
                 bo_xformed = bo.plasma_object.has_transform_animation
 
                 # MOUL: only objects that have animation data are kPhysAnim
-                if ver != pvMoul or bo_xformed:
+                if mod.enabled and (ver != pvMoul or bo_xformed):
```

Comment:
This could break animated clickables and regions because their physics objects are exported by nodes, not the collision modifier.